libs/portals/my-pages/health/src/screens/MedicineDelegation/MedicineDelegation.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import {
+  EmptyTable,
+  HEALTH_DIRECTORATE_SLUG,
+  IntroWrapper,
+  SortableTable,
+  m,
+} from '@island.is/portals/my-pages/core'
+import { useState } from 'react'
+import { messages } from '../../lib/messages'
+import DelegationModal from './components/DelegationModal'
+import { delegationData } from './utils/mockdata'
```

Comment:
Mockdata?

Reply:
Yes not ready for production, is under flag